tests/integration/test_github.py

```diff
@@ -153,7 +153,7 @@ def test_commit_flags(colin_project):
 
 
 def test_fork(colin_project_fork):
-    assert colin_project_fork.is_fork
+    assert colin_project_fork.is_fork is True
```

Comment:
oh, common

Reply:
this is on purpose: previously it returned instance of GitProject and I am making sure that it really is the True value

Reply:
`isinstance(colin_project_fork.is_fork, bool`)?

Reply:
why not check directly for True?